package/patterns-yast.spec

```diff
@@ -235,21 +235,23 @@ Provides:       pattern-visible()
 
 Recommends:     yast2-devtools
 Recommends:     yast2-testsuite
```

Comment:
really needed?

Reply:
Well, quite a lot of packages still have it in `BuildRequires` so still makes sense...

Reply:
ok